### PR TITLE
new file:   packages/batteries/batteries.3.9.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
Cc @kit-ty-kate ; sorry for the delay.

File: packages/batteries/batteries.3.9.0/opam [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "ocaml" {>= "4.05" & < "5.4"}
+  "camlp-streams"
+  "ocamlfind" {>= "1.5.3"}
+  "qtest" {with-test & >= "2.5"}
+  "qcheck" {with-test & >= "0.9" & < "0.14"}
+  "benchmark" {with-test & >= "1.6"}
+  "num"
+  "odoc" {with-doc}
+]
+conflicts: ["base-effects" "ocaml-option-no-flat-float-array"]
```
Comment (kit-ty-kate) [inline packages/batteries/batteries.3.9.0/opam:27]:
```suggestion
conflicts: ["ocaml-option-no-flat-float-array"]
```

**Comment by kit-ty-kate** [comment]
> Cc @kit-ty-kate ; sorry for the delay.

no problem!

**Comment by shonfeder** [comment]
Only some revdeps failing here, and none appear to be due to this change. Thank you for publishing the updates!

**Pull request merged**